internal/usage/usage.go

```diff
@@ -470,4 +470,5 @@ dbName and collection are required only for built-in roles.`
 	LogHostName                               = "Name of the host that stores the log files that you want to download."
 	requiredForAtlas                          = "Required when creating organizations authenticated with API Keys."
 	Decompress                                = "Flag that indicates whether to decompress the log files."
+	OnlineArchiveFilename                     = "Path to an optional JSON configuration file that defines online archive settings. To learn more about configuration files for the Atlas CLI, see https://www.mongodb.com/docs/atlas/cli/stable/reference/json/."
```

Comment:
https://dochub.mongodb.org/core/atlas-cli-reference-json

Reply:
@pierwill I'm surprised this dochub link is not named after Online Archive, you know that we will create a new page, right?

Reply:
Okay I think I can easily update the name. What do you suggest for the link name and target URL?

Reply:
https://dochub.mongodb.org/core/atlas-cli-json-online-archive-config ?

Reply:
https://dochub.mongodb.org/core/atlas-cli-json-online-archive-config ?

Reply:
> https://dochub.mongodb.org/core/atlas-cli-json-online-archive-config ?

@fmenezes So it will be:

link: https://dochub.mongodb.org/core/atlas-cli-json-online-archive-config
target: https://www.mongodb.com/docs/atlas/cli/stable/reference/json/

Is that right?

Reply:
Done. New dochub link: https://dochub.mongodb.org/core/atlas-cli-json-online-archive-config